src/datagrids/ImportDataTable.tsx

```diff
@@ -178,18 +204,44 @@ function ImportDataTable() {
   return (
     // IMPT: requires height and width for some reason?
     <div className="mx-auto" style={{ height: '40vh' }}>
-      <div className="ag-theme-alpine h-full w-full">
+      <div className="ag-theme-custom h-full w-full">
+        <Importheader />
+        <div className="grid-header">
+          {pageIndex * pageSize + 1}-{(pageIndex + 1) * pageSize} of {rowData.length} shown
+        </div>
         <AgGridReact
           rowData={rowData}
           columnDefs={columnDefs}
           defaultColDef={defaultColDef}
           onGridReady={onGridReady}
-          pagination
-          paginationPageSize={10}
           rowHeight={65}
           onCellClicked={onCellClicked}
           onGridSizeChanged={(e) => (e.clientWidth > 1024 ? e.api.sizeColumnsToFit() : null)}
         />
+        <div className="grid-bottom">
+          <div>
+            <span style={{marginRight: "10px"}}>Rows per page :</span> 
+            <select
```

Comment:
If we have to write our own pagination markup, it would be best to extract into its own component so its not copied over